works.yaml

```diff
@@ -1657,13 +1647,13 @@ OGRE:
   - http://sourceforge.net/p/ogre/code/HEAD/tree/trunk/COPYING
 OpenCV:
   uri:
-  - http://opencv.willowgarage.com/wiki/Welcome
+  - http://opencv.org
   tags:
   - graphics
   - software
   license:
   - Simplified BSD License
-  license_reference: []
+  license_reference: https://github.com/opencv/opencv/blob/master/LICENSE
```

Comment:
Please move this to a separate line, following the same pattern as other license reference entries.  This will probably actually break the works list page.